community/kernel/src/main/java/org/neo4j/storageengine/api/StoreReadLayer.java

```diff
@@ -78,6 +78,11 @@
     Iterator<NewIndexDescriptor> uniquenessIndexesGetAll();
 
     /**
+     * Returns all indexes (including unique) related to a property.
+     */
+    Iterator<NewIndexDescriptor> indexesAndUniqueIndexesRelatedToProperty( int propertyId );
```

Comment:
just to much existing naming schema can we rename it to `indexesGetForProperty` or `indexesGetAllForProperty`?